[2024-11-24T04:36:56.114+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-24T04:36:56.138+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_postgresql_2.load_attendance_data manual__2024-11-23T11:37:42.176780+00:00 [queued]>
[2024-11-24T04:36:56.147+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_postgresql_2.load_attendance_data manual__2024-11-23T11:37:42.176780+00:00 [queued]>
[2024-11-24T04:36:56.148+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 3
[2024-11-24T04:36:56.168+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): load_attendance_data> on 2024-11-23 11:37:42.176780+00:00
[2024-11-24T04:36:56.182+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=36071) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-24T04:36:56.181+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'api_to_postgresql_2', 'load_attendance_data', 'manual__2024-11-23T11:37:42.176780+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/api_to_pgdatabase.py', '--cfg-path', '/tmp/tmpweowvk3j']
[2024-11-24T04:36:56.183+0000] {standard_task_runner.py:72} INFO - Started process 36072 to run task
[2024-11-24T04:36:56.183+0000] {standard_task_runner.py:105} INFO - Job 46: Subtask load_attendance_data
[2024-11-24T04:36:56.243+0000] {task_command.py:467} INFO - Running <TaskInstance: api_to_postgresql_2.load_attendance_data manual__2024-11-23T11:37:42.176780+00:00 [running]> on host 91b3e7b0f6a8
[2024-11-24T04:36:56.377+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_to_postgresql_2' AIRFLOW_CTX_TASK_ID='load_attendance_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-23T11:37:42.176780+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-23T11:37:42.176780+00:00'
[2024-11-24T04:36:56.380+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-24T04:36:56.381+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-24T04:36:56.381+0000] {logging_mixin.py:190} INFO - Current task name:load_attendance_data state:running start_date:2024-11-24 04:36:56.139580+00:00
[2024-11-24T04:36:56.382+0000] {logging_mixin.py:190} INFO - Dag name:api_to_postgresql_2 and current dag run status:running
[2024-11-24T04:36:56.383+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-24T04:36:56.397+0000] {base.py:84} INFO - Retrieving connection 'postgres_default_2'
[2024-11-24T04:36:56.537+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-24T04:36:56.550+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-24T04:36:56.551+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=api_to_postgresql_2, task_id=load_attendance_data, run_id=manual__2024-11-23T11:37:42.176780+00:00, execution_date=20241123T113742, start_date=20241124T043656, end_date=20241124T043656
[2024-11-24T04:36:56.561+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-24T04:36:56.562+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-24T04:36:56.563+0000] {logging_mixin.py:190} INFO - Dag name:api_to_postgresql_2 queued_at:2024-11-23 11:37:42.587636+00:00
[2024-11-24T04:36:56.563+0000] {logging_mixin.py:190} INFO - Task hostname:91b3e7b0f6a8 operator:_PythonDecoratedOperator
[2024-11-24T04:36:56.605+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-24T04:36:56.635+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-24T04:36:56.638+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
